tox.ini

```diff
@@ -38,6 +38,8 @@ deps =
   sphinx
   -r{toxinidir}/docs/requirements-docs.txt
 commands = make -C docs html latex
+whitelist_externals =
+  make
```

Comment:
This just gets rid of a tox warning that complains about `make` not being part of the virtualenv.